### PR TITLE
SDIT-2337: ✨ Add in detail sections

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: assets/scss/application.scss [hunk 1]
```diff
@@ -11,4 +11,5 @@ $govuk-page-width: $moj-page-width;
 @import '@ministryofjustice/hmpps-connect-dps-components/dist/assets/header-bar';
 
 @import './components/header-bar';
+@import './components/summary-card';
```
Comment (petergphillips) [inline assets/scss/application.scss:14]: copied from prisoner profile

File: assets/scss/components/_summary-card.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.hmpps-summary-card {
```
Comment (petergphillips) [inline assets/scss/components/_summary-card.scss:1]: copied as is from prisoner profile

File: server/views/macros/summaryCardMacro.njk [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% macro summaryCard(params) %}
```
Comment (petergphillips) [inline server/views/macros/summaryCardMacro.njk:1]: copied from prisoner profile

File: server/views/macros/summaryListRow.njk [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{% macro summaryListRow(key, value, id, options) %}
```
Comment (petergphillips) [inline server/views/macros/summaryListRow.njk:1]: copied from prisoner profile

File: server/views/pages/detail.njk [hunk 5]
```diff
@@ -9,6 +9,77 @@
...
   <div class="govuk-grid-row govuk-body">
-    <h1>{{ detail.personalDetails.firstName | capitalize }} {{ detail.personalDetails.middleName | capitalize }} {{ detail.personalDetails.lastName }}</h1>
+    <div class="govuk-grid-column-three-quarters">
+      <h1>{{ personalDetails.firstName | capitalize }} {{ personalDetails.middleName | capitalize }} {{ personalDetails.lastName }}</h1>
+    </div>
   </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-quarter app-sidebar govuk-!-display-none-print">
+      <ul class="govuk-list hmpps-sidebar">
+        <li><a class="govuk-link govuk-link--no-visited-state" href="#summary">Subject</a></li>
```
Comment (petergphillips) [inline server/views/pages/detail.njk:19]: added all sections for now

File: server/views/partials/detail/summary.njk [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{% from "../../macros/summaryCardMacro.njk" import summaryCard %}
+{% from "../../macros/summaryListRow.njk" import summaryListRow %}
+
+{%- call summaryCard({title: "Subject", id: "summary"}) -%}
+  <div class="govuk-grid-row" data-qa="summary">
+    <div class="govuk-grid-column-full">
+      {% if personalDetails.dob %}
+        <dl class="govuk-summary-list govuk-summary-list--no-border govuk-!-margin-bottom-0">
+          {{ summaryListRow("Date of birth", (personalDetails.dob | formatDate) + " (" + personalDetails.dob | getAge + " years)", "dob") }}
```
Comment (petergphillips) [inline server/views/partials/detail/summary.njk:9]: start to add contents

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**